File: javascript/1160.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split("\n");
var q = parseInt(lines.shift());
for(let i = 1; i <= q; i++){
    let anos = 0;
    var cas = lines.shift().trim().split(" ");
    var PA = parseInt(cas.shift()), PB = parseInt(cas.shift());
    var G1 = parseFloat(cas.shift()), G2 = parseFloat(cas.shift());
    while(PA <= PB){
        PA += PA * (G1 / 100);
        PB += PB * (G2 / 100);
        PA = parseInt(PA);
        PB = parseInt(PB);
        anos++;
        if(anos > 100){
            break;
        }
    }
    console.log(anos > 100 ? "Mais de 1 seculo." : `${anos} anos.`);
}

File: C++/1160.cpp
#include <iostream>

using namespace std;

int main(){
    int n, i, PA, PB, anos;
    double GA, GB;
    cin >> n;
    
    for(i = 0; i < n; i++){
        
        cin >> PA >> PB >> GA >> GB;
        anos = 0;
        GA /= 100;
        GB /= 100;
        
        while(PA <= PB){
            PA += PA * GA;
            PB += PB * GB;
            anos++;
            if(anos > 100) break;
        }
        if(anos > 100) cout << "Mais de 1 seculo." << endl;
        else cout << anos << " anos." << endl;
    }

    return 0;
}